README.md

```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
     - [Websites](#websites)
 - [Contributing](#contributing)
 
+## Alternative Programming Languages
```

Comment:
I'm against the title, "Languages" would be sufficient and it wouldn't be placed as the first item...